### PR TITLE
Added how to retrieve invoices

### PR DESCRIPTION
**Pull request by mebverse**

### CODE REVIEW

File: retrieve-invoice.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.. _how-to-retrieve-invoice:
+
+How to retrieve invoices
+========================
+
+To retrieve your invoices, go to the organisation seetings of your project and select *invoices*.
```
Comment (evildmp) [inline retrieve-invoice.rst]: Spelling: settings

File: retrieve-invoice.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+.. image:: /images/organisation-settings.png
+   :alt: 'organisation settings'
+   :width: 300
+
+Here you will see the list of invoices, with the *date* issued, payment *status*, payment *period* and the *amount*
+paid. Click on the *invoice number* to view any other details corresponding to each invoice.
```
Comment (evildmp) [inline retrieve-invoice.rst]: Avoid using the word "click". Better to use "select", "choose", etc. (Reason: not everyone clicks, e.g. a user with a tablet will tap, etc.)

**Review by evildmp — CHANGES_REQUESTED** [review]

File: retrieve-invoice.rst (hunk 1 above)
Comment (mebverse) [reply retrieve-invoice.rst]: Done

File: retrieve-invoice.rst (hunk 2 above)
Comment (mebverse) [reply retrieve-invoice.rst]: Done

**Pull request merged**